packages/ibm-products/src/components/Tearsheet/next/Tearsheet.mdx

```diff
@@ -1,6 +1,309 @@
+  <Tearsheet.Header></Tearsheet.Header>
+
+  <Tearsheet.Influencer> </Tearsheet.Influencer>
+
+  <Tearsheet.Body>
+    <Tearsheet.MainContent></Tearsheet.MainContent>
+
+    <Tearsheet.SummaryContent></Tearsheet.SummaryContent>
+  </Tearsheet.Body>
+
+  <Tearsheet.Footer></Tearsheet.Footer>
+</Tearsheet>
+```
+
+On smaller screens or within narrow tearsheets, the Influencer and
+SummaryContent sections are hidden by default.
+
+However, they can be accessed through side panels: use the `influencerPanelOpen`
+and `onInfluencerPanelClose` props to control the left (influencer) panel, and
+the `summaryPanelOpen` and `onSummaryPanelClose` props to control the right
+(summary) panel.
+
+```jsx
+ <Tearsheet.Influencer
+    influencerPanelOpen={influencerPanelOpen}
+    onInfluencerPanelClose={() => setInfluencerPanelOpen(false)} >
+</Tearsheet.Influencer>
+
+<Tearsheet.SummaryContent
+    summaryPanelOpen={rightPanelOpen}
+    onSummaryPanelClose={() => setRightPanelOpen(false)}>
+</Tearsheet.Influencer>
+```
+
+## Tearsheet.Header
+
+Header sections consist of HeaderContent, HeaderActions and NavigationBar. <br/>
+**HeaderContent :**<br/> Here is where we render label, title, description,
+titleIcon . User can optionally render any custom content as children of
+HeaderContent **headerActions** The `headerActions` prop of `HeaderContent`
+accepts a `ReactNode` and occupies the right-hand area next to the header
+content. You can pass `<Tearsheet.HeaderActions>` to this prop, which positions
+itself at the top of the header—just before the AI label and close button. This
+area can contain action buttons or any other interactive elements. Each action
+item can be wrapped inside `Tearsheet.HeaderActionItem`, which automatically
+manages layout by collapsing overflowing actions into a menu button based on
+available space.<br/> **NavigationBar :**<br/> This section is anchored to the
+bottom of the header and can be used to render elements such as a tab bar or
+tags, depending on your needs. It also accepts a scroller prop, which takes a
+React node. To include a scroll button that expands or collapses the header,
+pass `PageHeader.ScrollButton` to the `scroller` prop.
+
+```jsx
+<Tearsheet.Header>
+  <Tearsheet.HeaderContent
+    {...headerContentProps}
+    headerActions={
+      <Tearsheet.HeaderActions>
+        <Tearsheet.HeaderActionItem>
+          <Button kind="tertiary" size="sm">
+            Action 1
+          </Button>
+        </Tearsheet.HeaderActionItem>
+        ...
+      </Tearsheet.HeaderActions>
+    }
+  ></Tearsheet.HeaderContent>
+  Any extra content like ProgressIndicator, notifications etc can go here
+  <Tearsheet.NavigationBar
+    scroller={<Tearsheet.ScrollButton />}
+  ></Tearsheet.NavigationBar>
+</Tearsheet.Header>
+```
+
+# Examples
+
+## Default
+
+<Canvas
+  of={stories.Default}
+  additionalActions={[
+    {
+      title: 'Open in Stackblitz',
+      onClick: () =>
+        stackblitzPrefillConfig({
```

Comment:
None of the "Open in stackblitz" work just yet, only because we haven't published `preview__Tearsheet` yet. Maybe comment out any `stackblitzPrefillConfig` integrations until we've published the preview, then we can add them back in.

I also noticed that some of the stackblitz's (outside of `preview__Tearsheet` not being available) are missing some variables that you will need to include via:

```ts
stackblitzPrefillConfig({
  customFunctionDefs: `const variant = ['wide', 'narrow', 'wide'];`
}),
```

In this case, it's the stacking example that needs `variant` defined.